src/flows/data_retention/delete_snowplow_events.sql

```diff
@@ -0,0 +1,5 @@
+-- Removes Snowplow Events (SNOWPLOW.ATOMIC.EVENTS) for deleted accounts
+delete from EVENTS as e
+using development.gaurang_data_retention_deletion.DELETED_USERS as d
```

Comment:
Looks reasonable to me! Is this table meant to eventually serve as the permanent record of user IDs that have been deleted?

Reply:
Yes. The `DELETED_USERS` and the `DELETED_EMAILS` table will hold the permanent record of user IDs and email IDs. They will exist in a separate protected DB/Schema (yet to be created and protected from deletions etc)

I will update the confluence doc with the information